extensions/import/src/wizard/pages/prosePreviewPage.ts

```diff
@@ -57,8 +57,8 @@ export class ProsePreviewPage extends ImportPage {
 			},
 			{
 				component: this.table,
-				title: '',
-				actions: [this.refresh]
+				title: ''
+				//actions: [this.refresh]  Removing this button until it makes sense (Issue #6611)
```

Comment:
I wouldn't remove this yet, it's not hurting anyone being there and some users might be changing their CSV files and don't want to go through the wizard every time they do that.

Reply:
Okay, makes sense I'll leave it there for now.